js2-mode.el

```diff
@@ -6163,8 +6163,6 @@ its relevant fields and puts it into `js2-ti-tokens'."
                              (setf (js2-token-beg token) (- js2-ts-cursor 2))
                              (js2-skip-line)
                              (setf (js2-token-comment-type token) 'line)
-                             ;; include newline so highlighting goes to end of window
```

Comment:
But then it's not the correct length, it's just a convenient length.

Could the +1 calculation be done at the time of font-locking when a node type is found to be a comment?